CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
   support for statically-linked libraries.
 * Improved examples
 * Corrected crashes and leaks in OOM scenarios
+* New API for sending commands to specific node
+* New API for node iteration, for sending commands to all nodes
```

Comment:
Iteration is not only for sending to all nodes. It can be used for sending to "some or all" (or maybe just to check which or how many nodes there are).

Reply:
corrected